app/src/main/res/layout/fragment_credit_card_editor.xml

```diff
@@ -94,86 +101,62 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:gravity="center_vertical"
+        android:letterSpacing="0.05"
         android:paddingStart="3dp"
         android:paddingEnd="0dp"
         android:text="@string/credit_cards_expiration_date"
         android:textColor="?primaryText"
         android:textSize="12sp"
-        android:layout_marginTop="10dp"
-        android:letterSpacing="0.05"
-        app:fontFamily="@font/metropolis_semibold" />
+        app:fontFamily="@font/metropolis_semibold"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/name_on_card_layout" />
 
-    <LinearLayout
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:orientation="horizontal"
-        android:baselineAligned="false">
-
-        <LinearLayout
-            android:layout_width="0dp"
-            android:layout_height="wrap_content"
-            android:layout_weight="1"
-            android:layout_marginEnd="24dp"
-            android:orientation="vertical">
-
-            <androidx.appcompat.widget.AppCompatSpinner
-                android:id="@+id/expiry_month_drop_down"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                tools:listitem="@android:layout/simple_list_item_1" />
 
-            <View
-                android:layout_width="match_parent"
-                android:layout_height="1dp"
-                android:background="?primaryText" />
```

Comment:
Leaving a note here, not to forget to add the view lines back in based on our offline conversation.